examples/fabric-admin/commands/clusters/ReportCommand.cpp

```diff
@@ -46,7 +46,7 @@ void ReportCommand::OnAttributeData(const app::ConcreteDataAttributePath & path,
 
     LogErrorOnFailure(RemoteDataModelLogger::LogAttributeAsJSON(path, data));
 
-    DeviceMgr().HandleAttributeChange(path, data);
+    DeviceMgr().HandleAttributeData(path, data);
```

Comment:
This redirects all attribute values. Please see device synchronization: we can use dedicated paths & loops instead of trying to hook in the main processing loop.

Reply:
Followup: https://github.com/project-chip/connectedhomeip/issues/34920